src/SonarScanner.MSBuild.PreProcessor/Protobuf/AnalysisCacheMsg.cs

```diff
@@ -0,0 +1,224 @@
+// <auto-generated>
```

Comment:
We have this section on sonar-dotnet that doesn't seem to be needed here. So we can remove it from there too?

https://github.com/SonarSource/sonar-dotnet/blob/638c7a6f4481677fc0c5c300d402d5d2aaeabad6/analyzers/src/SonarAnalyzer.Common/SonarAnalyzer.Common.csproj#L36-L39

Reply:
It's autogenerated code so we should not analyze it. That being said I was expecting a problem on the coverage side (and I also had a commit on top of these) but it seems the coverage is not reported for this file. My assumption is that it's handled by the coverlet.

Reply:
So we can delete the sonar-dotnet complicaton too?

Reply:
I hope so. I want to see this working end to end and if everything works find I will do the change also there,